### ISSUE TITLE
Ticks label not showing

### ISSUE THREAD

**Issue by manoj-netlings** [pimlie/vue-bootstrap-slider#2]
Hi i tried using the ticks labels but they are not showing below the slider . can you please guide me on this @pimlie

**Comment by pimlie** [comment]
The following principle should work:
```js
<template>
  <b-form-slider :ticks="my_ticks" :ticks-labels="my_tick_labels"></b-form-slider>
</template>

<script>
export default {
  data () {
    return {
      my_ticks: [1, 2, 3],
      my_tick_labels: ["One", "Two", "Three"]
    }
  }
}
</script>
```
If not, could you add an example maybe?

**Comment by manoj-netlings** [comment]
i have tried as you suggested @pimlie but it is not working i have created a plunker too please have a look http://plnkr.co/edit/TIQgp62QymaijzWDOLXT?p=preview

**Comment by pimlie** [comment]
This is a bug indeed, all bootstrap-slider properties that contain a `_` are not working. For use in vue those properties have been renamed to camelCase, but when passing them back to bootstrap-slider they are not renamed back from camelCase to snake_case

**Comment by pimlie** [comment]
Thanks for reporting, this should be fixed in v1.1.1

If your plunker is still not working then it has probably cached the @latest js file, if you change that to @1.1.1 it should work. Or at least this did it for me ;)

**Comment by manoj-netlings** [comment]
Thanks it is working now @pimlie
